test/java/src/test/java/ghostdriver/FileUploadTest.java

```diff
@@ -168,4 +171,96 @@ public void checkOnChangeEventIsFiredOnFileUpload() throws IOException {
 
         assertEquals("changed", result.getText());
      }
+
+    @Test
+    public void checkMultipleFileUploadCompletes() throws IOException {
```

Comment:
i trigger a native click event on the file element in `_postValueCommand`.  see ariya/phantomjs#14761 for more details on how this behaviour has changed.